src/main/java/org/ice4j/ice/Agent.java

```diff
@@ -1541,19 +1541,8 @@ protected void incomingCheckReceived(TransportAddress remoteAddress,
                                          String           localUFrag,
                                          boolean          useCandidate)
     {
-        if (isOver())
```

Comment:
Not really important so please feel free to ignore, but I was wondering if we could remove the if statement but keep and amend the comment?